backend/user-service/app/services/user.service.ts

```diff
@@ -0,0 +1,10 @@
+import { UserBody } from "../types/user-body";
+import { db } from "./db.service";
+
+export const userService = {
+    create: (body: UserBody) => db.user.create({ data: body }),
```

Comment:
i think that prisma generates the appropriate types for u to use - idt you have to create your own types

Reply:
Resolved 9c27c427960e34965a5305489676222a9aaccb7e